### ISSUE TITLE
Future plans?

### ISSUE THREAD

**Issue by fhunleth** [spawnfest/lasorex#1]
Hi! This isn't an issue, but I was wondering if you had plans to finish off your project now that Spawnfest is over? It would be super useful to have something like this on the Nerves project. We're currently using a non-interactive and bland version from https://github.com/fhunleth/toolshed - see midway down the README.md. Please let me know if you plan to do any more work on it and if so where you'd like to do it. Thanks!

**Comment by hoarf** [comment]
We would like to keep working on it. Right now the biggest issue is the performance of the ncurses NIF, we need to figure out why it is not able to show lots of processes.

**Comment by fhunleth** [comment]
I might be able to "help" you with that. `ex_ncurses` is a non-starter for Nerves since it only works on the console that you use to start Elixir. Most Nerves logins are via ssh.

I have a feeling like your use of `ex_ncurses` might be fairly minimal - like you need to know the terminal width/height, how to go to the top left, how to print in color, how to grab input without blocking, and maybe how to erase things. `IO.ANSI` has a lot of that support. I'm drawing a blank on the input part, but I know it's doable and that there are examples.

**Comment by hoarf** [comment]
Yeah, the first iteration was with IO.ANSI, but we switched to ncurses because of the input support. I will talk to the team and see if we can keep improving this. We are open to feature requests as well. :)
